src/index.ts

```diff
@@ -26,6 +29,12 @@ export interface Witness {
   proof: Buffer[];
 }
 
+export interface MultiWitness {
```

Comment:
Hm: So in the the last PR this seemed like a good idea because I thought we would cut down on duplication of nodes in the proof, but I don't think that's going to work.

The MultiWitness interface now seems sort of redundant now: I know that this is because the returned array has a different order than the input array, but if you just "unsort" the Witness array we'd probably have a cleaner interface. I guess you could preallocate the results array and insert the results by the index you calculate anyway.